src/components/CommentFeed.jsx

```diff
@@ -0,0 +1,88 @@
+import React, { Component } from 'react';
+import PropTypes from 'prop-types';
+import { API } from 'aws-amplify';
+import { Container, Spinner } from 'react-bootstrap';
+import CommentUpload from './CommentUpload';
+import CommentItem from './CommentItem';
+
+const propTypes = {
+  numComments: PropTypes.number,
+  videoID: PropTypes.number.isRequired,
+};
+
+const defaultProps = {
+  numComments: 20,
+};
+
+class CommentFeed extends Component {
+  constructor(props) {
+    super(props);
+    this.state = { comments: null };
+  }
+
+  async componentDidMount() {
+    const { videoID } = this.props;
+
+    try {
+      const comments = await API.get('videocloud', `/videos/${videoID}/comments`);
+
+      this.filterQuery(comments);
+    } catch (e) {
+      console.error(e);
+    }
+  }
+
+  compDate(a, b) {
+    return new Date(b.createdAt).getTime() - new Date(a.createdAt).getTime();
+  }
+
+  filterNewest(filtered) {
+    const { numComments } = this.props;
+    filtered.sort(this.compDate);
+    return filtered.filter((i, index) => (index < numComments));
+  }
+
+  filterQuery(comments) {
+    const { videoID } = this.props;
+    let filtered = comments;
+
+    filtered = filtered.filter(row => row.video === videoID);
```

Comment:
You can remove this now that the DB query only returns comments for "this" video.